controllers/mirrorpeer_controller.go

```diff
@@ -120,6 +121,9 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      tokenexchange.TokenExchangeName,
 				Namespace: mirrorPeer.Spec.Items[i].ClusterName,
+				Annotations: map[string]string{
+					addons.OwnerMirrorPeerKey: mirrorPeer.Name,
+				},
```

Comment:
there are many mirrorpeers. this probably won't work as expected.

Reply:
It works as expected but we don't need it now